app/_posts/2024-11-05-kuma-2-9-0.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Kuma 2.9 release with MeshTLS, Service redesign, GUI updates and more...
+description: Kuma 2.9 release with MeshTLS, Service redesign, GUI updates and more...
+date: 2024-11-06
+headerImage: /assets/images/blog/test-header.jpg
+tags:
+  - Release
+---
+
+We're excited to announce the biggest release of Kuma since 2.0! 
+In this release we offer a complete beta release of our service rework and namespaced policies.
+
+# Notable features
+In Kuma 2.9.x we've focused on 3 main areas:
+* `MeshTLS` which provides a more granular way to roll out strict Mutual TLS.
+* New resources `MeshService` and `MeshMultiZoneService` which provide a more intuitive and powerful way to manage services in your mesh.
+* A complete rework of our destination policy attachment which works with the new services. 
+
+This set of features greatly simplifies Kuma's user experience, by making it clearer how services exist and by attaching policies
+to specific resources. These changes are also fully integrated with our GUI thus making things incredible easy to understand and troubleshoot.
+
+We believe this to be a very first step in
+
+Feel free to check our [release notes](https://github.com/kumahq/kuma/releases/tag/2.9.0) for the full list of changes.
+
+## `MeshTLS`
+
+Mutual TLS is one of the most common use cases in Kuma. This is because it's the base on which you can build fully secured 0 trust environments.
```

Comment:
```suggestion
Mutual TLS is one of the most common use cases in Kuma. This is because it's the base on which you can build fully secured zero trust environments.
```